tests/restv1/RSVP/SingleTicketCest.php

```diff
@@ -239,7 +239,7 @@ public function should_hide_private_fields_to_public_queries(Restv1Tester $I) {
 			'rest_url'                      => $ticket_rest_url,
 			'provider'                      => 'rsvp',
 			'title'                         => $ticket_post->post_title,
-			'description'                   => $ticket_post->post_content,
+			'description'                   => $ticket_post->post_excerpt,
```

Comment:
This should be post_content, as per https://github.com/moderntribe/event-tickets/pull/1140